modules/internal/Atomics.chpl

```diff
@@ -308,7 +308,7 @@ module Atomics {
     var _v:atomic_bool = create_atomic_bool();
 
     pragma "no doc"
-    inline proc ~atomicbool() {
+    inline proc deinit() {
```

Comment:
I imagine so. The compiler inserts calls to deinitializers, right? If so, it is meaningful to talk about inlining at these call sites.